checklists/alz_checklist.en.json

```diff
@@ -668,6 +668,7 @@
             "guid": "143b16c3-1d7a-4a9b-9470-4489a8042d88",
             "id": "D01.03",
             "severity": "High",
+            "graph": "resources | where type =~ 'microsoft.network/publicIPAddresses' | extend properties = parse_json(properties) | project id, name, location, ddosProtectionPlan = properties.ddosSettings.protectionPlan | extend compliant = iff(isnotempty(ddosProtectionPlan), 1, 0) | where compliant == 1  project id, compliant",
```

Comment:
- I believe that there is a `|` missing after `compliant == 1` right?
- But more importantly, could you please remove the `where compliant == 1` altogether? Otherwise the non-compliant resources don't show up.